docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,226 @@
+https://github.com/hazelcast/hazelcast/pull/23471[#23471]
+* Fixed an issue where the unordered `mapUsingServiceAsync` was losing items when there are no watermarks.
+https://github.com/hazelcast/hazelcast/pull/23271[#23271]
+* Fixed an issue where the state of a job was being corrupted if a member is forcefully shut down and the job is restarted from snapshot.
+https://github.com/hazelcast/hazelcast/pull/23201[#23201]
+* Fixed an issue where disabling metrics and JMX on a Hazelcast client was failing the members to start.
+https://github.com/hazelcast/hazelcast/pull/23360[#23360]
+* Fixed an issue where a JDBC related exception message was mentioning data source reference instead of the table name.
+https://github.com/hazelcast/hazelcast/pull/23122[#23122]
+* Fixed an issue where the beginning time of a job was being recorded after the job is submitted, which was causing a race condition.
+https://github.com/hazelcast/hazelcast/pull/23065[#23065]
+* Fixed an issue where the `hz-cli` shell script was failing to run in bash on Windows systems since it uses a colon-separated `CLASSPATH`
+where each path is a Linux-style path. However, Java on Windows expects a semicolon-separated `CLASSPATH` where each path is a Windows-style path.
+As a consequence, the `hz-cli` could not be used from bash on Windows.
+https://github.com/hazelcast/hazelcast/pull/22884[#22884]
+* Fixed a memory leak due to incomplete clean-up of backup replica sync operations.
+https://github.com/hazelcast/hazelcast/pull/22769[#22769]
+* Fixed an issue where a map was loading old values with map.set() when map store offloading is enabled.
+https://github.com/hazelcast/hazelcast/pull/22602[#22602]
+* Fixed an issue where a CP Subsystem object could be destroyed without a permission even the object has security and permission configurations enabled.
+https://github.com/hazelcast/hazelcast/pull/22475[#22475]
+* Fixed an issue where external data store configurations could not be added dynamically.
+https://github.com/hazelcast/hazelcast/pull/22450[#22450]
+* Fixed an issue where replication over WAN was failing on the source cluster members, when there are multiple batch publishers configured in a single WAN replication.
+https://github.com/hazelcast/hazelcast/pull/22437[#22437]
+* Fixed an issue where the map entries recovered from persistence were not expiring after their time-to-live durations.
+https://github.com/hazelcast/hazelcast/pull/22279[#22279]
+* Fixed an issue where Hazelcast 5.1.x members in a Spring Boot application could not establish a connection to each other when deployed
+in Kubernetes cluster with Istio Envoy Proxy enabled.
+https://github.com/hazelcast/hazelcast/issues/22256[#22256]
+
+== Contributors
+
+We would like to thank the contributors from our open source community
+who worked on this release:
+
+* https://github.com/ivyanni[Ilia Vianni]
```

Comment:
we have more community contributions in this release: https://github.com/hazelcast/hazelcast/pulls?q=is%3Apr+milestone%3A5.3.0+is%3Aclosed+label%3A%22source%3A+community%22+-author%3Aapp%2Fdependabot